example/rest/orders-walkthrough/app/app.go

```diff
@@ -0,0 +1,35 @@
+package app
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/z5labs/humus/example/rest/orders-walkthrough/endpoint"
+	"github.com/z5labs/humus/example/rest/orders-walkthrough/service"
+	"github.com/z5labs/humus/rest"
+
+	"go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp"
+)
+
+// Init initializes the REST API with all endpoints and services.
+func Init(ctx context.Context, cfg Config) (*rest.Api, error) {
+	// Create OTel-instrumented HTTP client for service calls
+	httpClient := &http.Client{
+		Transport: otelhttp.NewTransport(http.DefaultTransport),
+	}
+
+	// Initialize services
+	dataSvc := service.NewDataService(cfg.Services.DataURL, httpClient)
+	restrictionSvc := service.NewRestrictionService(cfg.Services.RestrictionURL, httpClient)
+	eligibilitySvc := service.NewEligibilityService(cfg.Services.EligibilityURL, httpClient)
+
+	// Create API with endpoints
+	api := rest.NewApi(
+		"Orders API",
+		"v1.0.0",
```

Comment:
this should come from `Config`
```suggestion
		cfg.OpenApi.Version,
```